.azure/pipelines/ci.yml

```diff
@@ -264,12 +264,21 @@ stages:
         -pack
         -noBuildNodeJS
         -noBuildJava
+        -binaryLog
         /p:DotNetSignType=$(_SignType)
         /p:OnlyPackPlatformSpecificPackages=true
         /p:AssetManifestFileName=aspnetcore-win-arm.xml
         $(_BuildArgs)
         $(_PublishArgs)
         $(_InternalRuntimeDownloadArgs)
+      afterBuild:
```

Comment:
I'm of two minds about making this part and the new artifact upload conditional on being an official build of 'main'. On one hand, the copy and the artifact upload take only a bit more than a minute to perform i.e. they aren't slowing the build down. On the other, nobody but the `SourceIndexStage1` job would want to look at the artifact (once this PR is in). Thoughts @dotnet/aspnet-build❔

Reply:
I don't see a downside to conditioning this on the build being against `main` - a minute is more than a negligible amount of time